src/MergeContext.js

```diff
@@ -351,7 +364,7 @@ class MergeContext {
             requiredContexts = await GH.getProtectedBranchRequiredStatusChecks(this._prBaseBranch());
         } catch (e) {
            if (e.name === 'ErrorContext' && e.notFound())
-               this._log("required status checks not found");
+               Log.LogExceptionMessage(e, this._toString() + " required status checks not found");
```

Comment:
AFAICT, what we want to say here is "no status checks are required". In other words, this is not an error (unlike the implication of "X not found"), and that is why we do not treat it as an error.

Reply:
Done.